infra/growth/src/util/arrayUtils.js

```diff
@@ -0,0 +1,9 @@
+function concat(x, y) {
+  return x.concat(y)
+}
+
+function flatMap(f, xs) {
+  return xs.map(f).reduce(concat, [])
```

Comment:
Just a note. We can probably skip the `.map` here.
```
xs.reduce((a, d) => [...a, ...f(d)], [])
```
But your approach looks pretty cool.

Reply:
Thanks! Skipping the concat function like suggested. 

About the `map` I kind of prefer keeping my version. While your works nicely and uses less code I think with `map` the code is a bit more readable.